content/account/dashboard/console.textile

```diff
@@ -0,0 +1,80 @@
+---
+title: Dev console
+meta_description: "Gain realtime insights into application-wide events, such as connection status changes, channel activity, and event logs.”
+meta_keywords: “Ably dev console, realtime monitoring, connection status changes, channel activity, event logs"
+---
+
+The dev console tab provides realtime insights into application-wide events, such as connection status changes and channel activity. These features enable you to:
+
+* Observe changes in connection status across the application.
+* View activity on specific channels, including message traffic and channel events.
+* Examine event logs to troubleshoot your application.
+
+h2. Application-wide events interface
+
+The application-wide events interface allows you to monitor your application's health and activity in realtime.
+
+The following explains the realtime monitoring tools in the application-wide events interface:
+
+|_. Field |_. Description |
+| *API key* | The API key associated with your Ably application with an additional filter. |
```

Comment:
I *think* this is the API key that you're using to view events. Though worth checking that assumption.

Reply:
True :)